util/src/main/java/org/kaazing/gateway/util/GL.java

```diff
@@ -20,6 +20,8 @@
 
 public class GL {
 
+    public static final String CLUSTER_LOGGER_NAME = "ha";
```

Comment:
That was just Chris Barrows suggestion when I talked to him.  I think the cluster context interface makes more sense anyways as ha really has to do with that